src/Sulu/Bundle/AdminBundle/Command/UpdateBuildCommand.php

```diff
@@ -268,23 +273,46 @@ private function doManualBuild(SymfonyStyle $ui): int
             return static::EXIT_CODE_ABORTED_MANUAL_BUILD;
         }
 
-        $ui->title('Start manual build ...');
+        $conflictingDirectories = [];
+        foreach ([
+             $this->projectDir . '/node_modules',
+             \dirname(__DIR__, 5) . '/node_modules',
+        ] as $conflictingDirectory) {
+            $renamedConflictingDirectoryName = $conflictingDirectory . '.bak';
+            if ($this->filesystem->exists($conflictingDirectory)) {
+                $this->filesystem->rename($conflictingDirectory, $renamedConflictingDirectoryName);
+            } elseif (!$this->filesystem->exists($renamedConflictingDirectoryName)) {
+                // only continue here when also the renamed node_modules does not exist from a previous run
+                continue;
+            }
+
+            $ui->writeln(\sprintf('Renaming conflicting directory "%s"', $conflictingDirectory));
```

Comment:
```suggestion
            $ui->writeln(\sprintf('Renaming directory "%s" to prevent webpack resolution conflicts during build', $conflictingDirectory));
```